### DISCUSSION TITLE
Is it possible to use GeoPandas on Windows 7 ?

### DISCUSSION THREAD

**Discussion by bill-ion** [geopandas/geopandas#3218] [Installation] [answered]
- [x] I have checked that this issue has not already been reported.

- [x] I have confirmed this bug exists on the latest version of geopandas.

- [x] (optional) I have confirmed this bug exists on the main branch of geopandas.

---

**Note**: Please read [this guide](https://matthewrocklin.com/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.

#### Code Sample, a copy-pastable example

```python
import geopandas as gpd

```

#### Problem description
I got the following error when I import GeoPandas in PyCharm on Windows 7:
```
C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\site-packages\geopandas\_compat.py:115: UserWarning: The Shapely GEOS version (3.11.3-CAPI-1.17.3) is incompatible with the GEOS version PyGEOS was compiled with (3.10.4-CAPI-1.16.2). Conversions between both will be slow.
  shapely_geos_version, geos_capi_version_string
Traceback (most recent call last):
  File "path/to/my_script.py", line 1, in <module>
    import geopandas as gpd
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\site-packages\geopandas\__init__.py", line 7, in <module>
    from geopandas.io.file import _read_file as read_file  # noqa
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\site-packages\geopandas\io\file.py", line 14, in <module>
    import fiona
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\site-packages\fiona\__init__.py", line 46, in <module>
    _delvewheel_patch_1_5_1()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\site-packages\fiona\__init__.py", line 43, in _delvewheel_patch_1_5_1
    raise OSError('Error loading {}; {}'.format(lib, ctypes.FormatError()))
OSError: Error loading gdal-fd9ba3209f601e85fe39e4d1757194df.dll; The specified procedure could not be found
```
If I do it in CMD, it will first popup a window showing the following error:
```
The procedure entry point "GetSystemTimePreciseAsFileTime" could not be located in the dynamic link library KERNEL32.dll
```
After I closed the window the same OSError would be shown in CMD again.

I tried it on a different Windows 7 machine and the error persists. On Windows 10 and Windows 11 evething is normal.

My OS: Windows 7
My Python version: 3.7.9
The related modules version:
- attrs-23.2.0-py3-none-any.whl
- certifi-2024.2.2-py3-none-any.whl
- click-8.1.7-py3-none-any.whl
- click_plugins-1.1.1-py2.py3-none-any.whl
- cligj-0.7.2-py3-none-any.whl
- colorama-0.4.6-py2.py3-none-any.whl
- fiona-1.9.5-cp37-cp37m-win_amd64.whl
- geopandas-0.10.2-py2.py3-none-any.whl
- importlib_metadata-6.7.0-py3-none-any.whl
- numpy-1.21.6-cp37-cp37m-win_amd64.whl
- pandas-1.1.5-cp37-cp37m-win_amd64.whl
- pyproj-3.2.1-cp37-cp37m-win_amd64.whl
- python_dateutil-2.9.0.post0-py2.py3-none-any.whl
- pytz-2024.1-py2.py3-none-any.whl
- setuptools-68.0.0-py3-none-any.whl
- shapely-2.0.3-cp37-cp37m-win_amd64.whl
- six-1.16.0-py2.py3-none-any.whl
- typing_extensions-4.7.1-py3-none-any.whl
- zipp-3.15.0-py3-none-any.whl

#### Expected Output
Normally import GeoPandas.

#### Output of ``geopandas.show_versions()``

<details>

(Could not import geopandas)

</details>

**Comment by m-richards** [comment, marked as answer]
I think this is out of scope for the project - windows 7 reached end of life in 2020, and the last version of geopandas to support python 3.7 v 0.10.2 was released in October 2021.

Having said that, you could try using an older versions of fiona to see if that makes a difference (1.9.5 is from 2023, 1.8.20 would have been the version around when 0.10 was released). Since you're on windows you  may want to find a wheel binary for fiona and its dependencies from https://www.lfd.uci.edu/~gohlke/pythonlibs/ rather than build from source).

**Reply by bill-ion** [reply]
Installing the older version of fiona (==1.9.4) works for me. Thanks for your very helpful answer!
